tutoraspects/plugin.py

```diff
@@ -312,13 +313,26 @@
 # For each task added to MY_INIT_TASKS, we load the task template
 # and add it to the CLI_DO_INIT_TASKS filter, which tells Tutor to
 # run it as part of the `init` job.
-for service, template_path, priority in MY_INIT_TASKS:
-    full_path: str = pkg_resources.resource_filename(
-        "tutoraspects", os.path.join("templates", *template_path)
-    )
-    with open(full_path, encoding="utf-8") as init_task_file:
-        init_task: str = init_task_file.read()
-    hooks.Filters.CLI_DO_INIT_TASKS.add_item((service, init_task), priority=priority)
+try:
+    for service, template_path, priority in MY_INIT_TASKS:
+        full_path: str = pkg_resources.resource_filename(
+            "tutoraspects", os.path.join("templates", *template_path)
+        )
+        with open(full_path, encoding="utf-8") as init_task_file:
+            init_task: str = init_task_file.read()
+        hooks.Filters.COMMANDS_INIT.add_item(
+            (service, init_task)
+        )  # pylint: disable=no-member
+except AttributeError as e:
```

Comment:
Can you comment why we're doing these in both places?

Reply:
I think we still need a comment here explaining why this try/except is needed

Reply:
Done